### DISCUSSION TITLE
Manual Vercel Installation Instructions for Hono Support

### DISCUSSION THREAD

**Discussion by mmurrs** [wevm/frog-archived#252] [Ideas]
Adding Vercel manually to the base Frog template, under the Devtool section, the docs include:

 if (import.meta.env?.MODE === 'development') devtools(app, { serveStatic })
else devtools(app, { assetsPath: '/.frog' })

Which the import.meta is only supported in Vite and not hono. Request for the instructions to include the proper snippet for Vercel integration in Hono.
Docs: https://frog.fm/platforms/vercel#setup-devtools
![image](https://github.com/wevm/frog/assets/37455908/b68a843e-378e-4cf3-8884-7865bdf53132)

**Comment by dalechyn** [comment]
Transferred to an issue https://github.com/wevm/frog/issues/487
